server/src/routes/api/v1/boardGameReviewsRouter.js

```diff
@@ -0,0 +1,22 @@
+import express from "express";
+import {Review} from "../../../models/index.js"
+
+import ReviewSerializer from "../../../serializers/ReviewSerializer.js"
+
+const boardGameReviewsRouter = new express.Router({mergeParams: true})
+
+boardGameReviewsRouter.get("/", async (req, res) => {
```

Comment:
we would need this nested router when we are adding a new review for a board game. To get all of the reviews for a particular board game should be handled in the board game router in the show action when we query for the game, we can use a relatedquery for the associated reviews